lib/core/test/integration/Test/Integration/Scenario/API/Wallets.hs

```diff
@@ -134,8 +133,8 @@ spec = do
     it "WALLETS_CREATE_02 - Restored wallet preserves funds" $ \ctx -> do
         wSrc <- fixtureWallet ctx
         -- create wallet
-        Stdout mnemonics <- generateMnemonicsViaCLI []
-        let payldCrt = payloadWith "!st created" (T.words $ T.pack mnemonics)
+        mnemonics <- mnemonicToText @15 . entropyToMnemonic <$> genEntropy
+        let payldCrt = payloadWith "!st created" mnemonics
```

Comment:
I've "inlined" this CLI call since there's no reason the API scenarios would require something from the CLI (and this complexifies calling these specs for now reasons). We do have dedicated CLI specs for testing mnemonic generation so, all good.